runtime/middleware/jwt.go

```diff
@@ -0,0 +1,95 @@
+package middleware
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/hypermodeinc/modus/runtime/config"
+	"github.com/hypermodeinc/modus/runtime/logger"
+	"github.com/hypermodeinc/modus/runtime/utils"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type JWTClaimsKey string
+
+const JWTClaims JWTClaimsKey = "jwt_claims"
+
+func HandleJWT(next http.Handler) http.Handler {
```

Comment:
```suggestion
func HandleJWT(next http.Handler) http.Handler {
	jwtParser := jwt.NewParser()
```

You can then reuse the `jwtParser` instance on every request when parsing, for both lines 33 and 58.